packages/sites/genomics-site/webapp/wdkCustomization/js/client/components/questions/EdaNotebookQuestionForm.tsx

```diff
@@ -0,0 +1,53 @@
+import DefaultQuestionForm, {
+  Props,
+} from '@veupathdb/wdk-client/lib/Views/Question/DefaultQuestionForm';
+import React, { useCallback } from 'react';
+import { EdaNotebookParameter } from './EdaNotebookParameter';
+import {
+  ParameterGroup,
+  Parameter,
+  ParameterValues,
+} from '@veupathdb/wdk-client/lib/Utils/WdkModel';
+
+export const EdaNotebookQuestionForm = (props: Props) => {
+  const { searchName } = props;
+  if (!searchName) {
+    throw new Error('No search defined.');
+  }
+
+  // We'll use this function throughout the notebook to update any wdk parameters.
+  const wdkUpdateParamValue = useCallback(
+    (
+      parameter: Parameter,
+      newParamValue: string,
+      paramValues: ParameterValues
+    ) => {
+      props.eventHandlers.updateParamValue({
+        searchName,
+        parameter,
+        paramValues,
+        paramValue: newParamValue,
+      });
+    },
+    [props, searchName]
+  );
+
+  // An override that renders the notebook instead of any default parameter or parameter group ui.
+  const renderParamGroup = (group: ParameterGroup, formProps: Props) => {
```

Comment:
My IDE is warning about unused args `group` and `formProps`.  If I prefix with `_` the warning goes away. This also seems to compile fine:

```